apps/frontend/public/static/export/template.html

```diff
@@ -0,0 +1,294 @@
+            document.body.removeChild(a);
+          }, 0)
+        }
+      }
+      
+      /** Converts a string to an array buffer */
+      function s2ab(s) {
+        const buf = new ArrayBuffer(s.length); //convert s to arrayBuffer
+        const view = new Uint8Array(buf); //create uint8array as viewer
+        for (let i = 0; i < s.length; i++) {
+          view[i] = s.charCodeAt(i) & 0xff; //convert to octet
+        }
+        return buf;
+      }
+    </script>
+  </head>
+  <body>
+    <nav class="navbar fixed-top navbar-expand-lg navbar-light bg-light">
+      <div class="container-fluid">
+        <a class="navbar-brand" href="#">Heimdall Report</a>
+        <button class="navbar-toggler" type="button" data-bs-toggle="collapse" data-bs-target="#navbarSupportedContent" aria-controls="navbarSupportedContent" aria-expanded="false" aria-label="Toggle navigation"> <span class="navbar-toggler-icon"></span> </button>
+        <div class="collapse navbar-collapse" id="navbarSupportedContent">
+          <ul class="navbar-nav me-auto mb-2 mb-lg-0">
+            <li class="nav-item"> <a class="nav-link active" aria-current="page" href="#">Profile Info</a> </li>
+          </ul>
+          <div class="d-flex">
+            <a id="generate" onclick="$(this).attr('href', 'data:text/plain;base64,' + utf8_to_b64($('html').clone().find('#generate').remove().end()[0].outerHTML));" download="report.html">
+              <svg style="width:24px;height:24px" viewBox="0 0 24 24" onclick="savePage()">
+                <path fill="currentColor" d="M5,20H19V18H5M19,9H15V3H9V9H5L12,16L19,9Z" />
+              </svg>
+            </a>
+          </div>
+        </div>
+      </div>
+    </nav>
+    <div class="container-fluid" style="margin-top: 80px;">
+      <div class="card bg-light">
+        <div class="card-body">
+          <h5 class="card-title">Profile info</h5>
+          <div id="profileInfos" class="row">
+            {{#files}}
+            <div class="col-xl results-set">
+              <span>Filename: <strong>{{filename}}</strong></span><br />
+              <span>Tool Version: <strong>{{toolVersion}}</strong></span><br />
+              <span>Platform: <strong>{{platform}}</strong></span><br />
+              <span>Duration: <strong>{{duration}}</strong></span><br />
+            </div>
+            {{/files}}
+          </div>
+        </div>
+      </div>
+    </div>
+    <div class="container-fluid pt-4" id="statusCards">
+      <div class="row">
+        <div class="col-md">
+          <div class="card bg-success tall-card">
+            <div class="card-body">
+              <div class="card-title">
+                <div class="h4">
+                  <svg style="width:24px; height:24px" viewBox="0 0 24 24">
+                    <path fill="white" d="M12 2C6.5 2 2 6.5 2 12S6.5 22 12 22 22 17.5 22 12 17.5 2 12 2M10 17L5 12L6.41 10.59L10 14.17L17.59 6.58L19 8L10 17Z" />
+                  </svg>
```

Comment:
Can we just bundle @mdi/font similar to how we are bundling bootstrap so that it is more obvious for future developers that these are icons?